build.gradle

```diff
@@ -44,7 +44,7 @@ subprojects {
     }
   }
 
-  tasks.withType(JavaCompile) {
+  tasks.withType(JavaCompile).configureEach {
```

Comment:
interesting that this was working before...

Reply:
🤷‍♂️